horovod/run/horovod_task_fn.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Uber Technologies, Inc. All Rights Reserved.
```

Comment:
Should we have a similar structure in this folder - one folder for tasks, one for the driver?